src/app/facility-descripton-mobile/facility-descripton-mobile.component.scss

```diff
@@ -0,0 +1,10 @@
+@media (max-width: 375px) {
+    span {
+        overflow: scroll;
+    }
+}
+.image-wrapper
+{
+    width: 375px;
+    padding: 0;
+}
```

Comment:
Drobna uwaga, jeżeli wszędzie w kodzie trzymasz się jednego sposobu, czyli po znaczniku dajesz spacje a potem nawias to trzymaj się tego, a nie że tutaj nagle pojawia się enter